src/promptflow/tests/sdk_cli_test/e2etests/test_cli.py

```diff
@@ -56,6 +57,37 @@ def run_pf_command(*args, cwd=None):
         os.chdir(origin_cwd)
 
 
+def run_batch(local_client, line_timeout_seconds):
+    os.environ["PF_LINE_TIMEOUT_SEC"] = line_timeout_seconds
+    run_id = str(uuid.uuid4())
+    run_pf_command(
+        "run",
+        "create",
+        "--flow",
+        f"{FLOWS_DIR}/simple_flow_with_ten_inputs",
+        "--data",
+        f"{FLOWS_DIR}/simple_flow_with_ten_inputs/data.jsonl",
+        "--name",
+        run_id,
+    )
+    run = local_client.runs.get(name=run_id)
+    local_storage = LocalStorageOperations(run)
+    detail = local_storage.load_detail()
+    flow_runs_list = detail["flow_runs"]
+    for i, flow_run in enumerate(flow_runs_list):
+        if line_timeout_seconds == "54":
```

Comment:
Using `== "54"` and ` != 9` may not be a good practice to do these check.
Maybe we can have another parameter `expected_failed_line_numbers`

Reply:
Add time_out_index